llm_transparency_tool/models/tlens_model.py

```diff
@@ -43,16 +42,21 @@ def load_hooked_transformer(
     #     n_devices = torch.cuda.device_count()
     # else:
     #     n_devices = 1
-    tlens_model = transformer_lens.HookedTransformer.from_pretrained(
-        model_name,
-        hf_model=hf_model,
-        fold_ln=False,  # Keep layer norm where it is.
-        center_writing_weights=False,
-        center_unembed=False,
-        device=tlens_device,
-        # n_devices=n_devices,
-        dtype=dtype,
-    )
+    
+    if hf_model is not None:
+        tlens_model = hf_model_to_hooked_transformer(hf_model, tlens_device, dtype)
```

Comment:
I did a monkey patch to the upstream `TransformerLens` for this. For the separation of concern, support more model arch is the responsibility of that library. `LLM-TT` should deal with right configuration to load from local directory and accepting custom model arch if that is accepted by `TransformerLens`.